src/OpenLoco/Graphics/Colour.cpp

```diff
@@ -48,6 +48,40 @@ namespace OpenLoco::Colours
         ExtColour::translucentMutedRed1,
     };
 
+    static constexpr std::array<ExtColour, 31> _shadowColourMap = {
```

Comment:
Unsure if its just the first couple entries or the whole thing. Needs further investigation.